### DISCUSSION TITLE
Mass deployment

### DISCUSSION THREAD

**Discussion by HCjoy** [iterate-ch/cyberduck#16190] [Q&A]
I need to deploy Mountain Duck in large quantities to Windows and Mac systems and configure each instance with a private access key and secure access key. Does anyone have a method for this? (The environment already has Active Directory)

**Comment by dkocher** [comment]
You will need to 

- Deploy a `.duck` bookmark file to the _Bookmarks_ folder in the [application support directory](https://docs.mountainduck.io/mountainduck/support/#application-support-folder). 
- Add credentials for the access key matching `Username` in the bookmark file to the login keychain (macOS) or credentials manager (Windows) respectively.
